lib/hyrax/specs/shared_specs/indexers.rb

```diff
@@ -18,7 +18,8 @@
   describe '#to_solr' do
     it 'indexes alternate_ids' do
```

Comment:
if we're changing the scope of this test, we should also change its description.